setup.sh

```
@@ -0,0 +1,52 @@
+# configファイルの実体はリポジトリで管理できるように、ホームディレクトリ側はシンボリックリンクを貼るつくりにしておく。
+# NOTE:
+# Nixを併用することでプロジェクトの開発環境に入った時のみリンクを切り替えるつくりも可能になりそう。
+
+# opencode.jsonc
+OPCODE_DIR=".config/opencode"
+OPCODE_FILE="opencode.jsonc"
+OPCODE_SRC="$(pwd)/$OPCODE_DIR/$OPCODE_FILE"
+OPCODE_DEST="$HOME/$OPCODE_DIR/$OPCODE_FILE"
+
+LINK_DIR="$HOME/$TARGET_DIR"
+LINK_PATH="$LINK_DIR/$TARGET_FILE"
```

Shellcheck (0.11.0)

[warning] LINK_PATH appears unused. Verify use (or export if used externally).

(SC2034)